src/controllers/product-controller.js

```diff
@@ -1,4 +1,79 @@
-// eslint-disable-next-line no-unused-vars
+const config = require('config');
+const FileType = require('file-type');
+const Promise = require('bluebird');
+
 const { ProductModel } = require('../models');
+const S3Lib = require('../libs/s3-lib');
+const NotFoundError = require('../errors/not-found-error');
+const ForbiddenError = require('../errors/forbidden-error');
```

Comment:
these errors are re-exported in '../errors' index file with their names, so we can destruct them from 1 file, instead of listing each error require on separate line

const { NotFoundError, ForbiddenError  }  = require('../errors');